### PR TITLE
MODSOURMAN-170: setup isbn type id from settings

### PR DESCRIPTION
**Pull request by OleksiiKuzminov**
ISBN
Comes from 020 $a $q $c all put together in whatever order they appear in the 020; $q may be repeated; remove the subfield mark and letter; add a space between subfields

Invalid ISBN 
Comes from 020 $z $q $c all put together in whatever order they appear in the 020; $q may be repeated; remove the subfield mark and letter; add a space between subfields

### CODE REVIEW

File: mod-source-record-manager-server/src/main/resources/rules.json [hunk 1]
```diff
@@ -359,8 +359,14 @@
           ],
```
Comment (Igor-Gorchakov) [inline mod-source-record-manager-server/src/main/resources/rules.json]:
```suggestion
          "q",
          "c"
          ],
```

Comment (Igor-Gorchakov) [inline mod-source-record-manager-server/src/main/resources/rules.json]: subfields :  
"a", "c", "q"   ?

File: mod-source-record-manager-server/src/main/resources/rules.json [hunk 2]
```diff
@@ -359,8 +359,14 @@
...
           "rules": [
```
Comment (Igor-Gorchakov) [inline mod-source-record-manager-server/src/main/resources/rules.json]: "applyRulesOnConcatenatedData": true

File: mod-source-record-manager-server/src/main/resources/rules.json (hunk 1 above)
Comment (OleksiiKuzminov) [reply mod-source-record-manager-server/src/main/resources/rules.json]: yes

File: mod-source-record-manager-server/src/main/resources/rules.json [hunk 3]
```diff
@@ -398,8 +404,14 @@
           ],
```
Comment (Igor-Gorchakov) [inline mod-source-record-manager-server/src/main/resources/rules.json]: subfields: "z",
            "q",
            "c"   ?

Comment (Igor-Gorchakov) [reply mod-source-record-manager-server/src/main/resources/rules.json]: "applyRulesOnConcatenatedData": true,

**Pull request merged**